### ISSUE TITLE
Victorialogs : a tail query no longer works with version 0.29.0

### ISSUE THREAD

**Issue by crocchietta** [VictoriaMetrics/VictoriaMetrics#7028]
### Is your question request related to a specific component?

Victorialogs

### Describe the question in detail

With version 0.28.0, the following query worked:

`curl -snkN --request POST --url https://@ip:9428/select/logsql/tail -d "query=job:fail2ban_log (level:INFO or level:NOTICE) service:spb (Ban or Unban) _time:5m"`

This same query no longer returns anything with version 0.29.0. 

If I remove the time:5m filter, it works fine :

`curl -snkN --request POST --url https://@ip:9428/select/logsql/tail -d "query=job:fail2ban_log (level:INFO or level:NOTICE) service:spb (Ban or Unban)"`

I use the _time filter with reference to the documentation : `it must select _time field`.

Do I misunderstand the documentation ?

Is my query poorly constructed?

### Troubleshooting docs

- [ ] General - https://docs.victoriametrics.com/troubleshooting/
- [ ] vmagent - https://docs.victoriametrics.com/vmagent/#troubleshooting
- [ ] vmalert - https://docs.victoriametrics.com/vmalert/#troubleshooting

**Comment by yuriydzobak** [comment]
Hi VM team,
I faced with tail issue too
It stopped worked in 0.29.0
This query post logs but then it stopped and don’t print new logs
```
curl https://localhost:9428/select/logsql/tail -N -d 'query=_time:1m'
```
it doesn’t work at all, any request if you added some filtering
```
curl https://localhost:9428/select/logsql/tail -N -d 'query=_time:1m pod_name=grafana'
or
curl https://localhost:9428/select/logsql/tail -N -d 'query=_time:1m grafana'
or
curl https://localhost:9428/select/logsql/tail -N -d 'query=_time:1m _msg:grafana'
or
curl http://localhost:9428/select/logsql/tail -N -d 'query=_time:1m _stream_id=0000000000000000fd1fa93d4952b503b6527143ba34db72'
```
and no logs about issue in VictoriaLogs or in curl
```
curl http://localhost:9428/select/logsql/tail -N -d 'query=_time:1m _stream_id=0000000000000000fd1fa93d4952b503b6527143ba34db72' -v
* Host localhost:9428 was resolved.
* IPv6: ::1
* IPv4: 127.0.0.1
*   Trying [::1]:9428...
* Connected to localhost (::1) port 9428
> POST /select/logsql/tail HTTP/1.1
> Host: localhost:9428
> User-Agent: curl/8.7.1
> Accept: */*
> Content-Length: 74
> Content-Type: application/x-www-form-urlencoded
>
* upload completely sent off: 74 bytes
```
Also, I’ve tried with -x POST too

**Comment by valyala** [comment]
@crocchietta and @yuriydzobak , thanks for the bugreport!

This bug should be fixed in the commit 255d1d4e139acc0bc66cfbec2ecdaffdf9177e14 . This commit will be included in the next release. In the mean time you can build VictoriaLogs from this commit according to [these docs](https://docs.victoriametrics.com/victorialogs/quickstart/#building-from-source-code) and verify whether the bug is fixed there.

**Comment by valyala** [comment]
The bugfix for this issue has been included in [v0.30.0-victorialogs](https://docs.victoriametrics.com/victorialogs/changelog/) release. Closing the issue as fixed then.
